tools/server-spring/src/test/java/org/eclipse/rdf4j/http/server/repository/statements/TestExportStatementsView.java

```diff
@@ -61,6 +60,6 @@ public void shouldReturnSC_INTERNAL_SERVER_ERRORIfExceptionIsThrown() throws Exc
 		}
 
 		Assert.assertNotNull(exception);
-		Assert.assertThat(exception.getMessage(), CoreMatchers.containsString(REPOSITORY_ERROR_MSG));
+		Assert.assertTrue(exception.getMessage().contains(REPOSITORY_ERROR_MSG));
```

Comment:
This isn't actually an AssertJ assertion, but this is fine too - thanks for removing the Hamcrest matcher